redbot/core/core_commands.py

```diff
@@ -1117,17 +1117,18 @@ async def embedset(self, ctx: commands.Context):
         The default is to use embeds.
 
         The embed settings are checked until the first True/False in this order:
-            - In guild context:
-                1. Channel override - `[p]embedset channel`
-                2. Server command override - `[p]embedset command server`
-                3. Server override - `[p]embedset server`
-                4. Global command override - `[p]embedset command global`
-                5. Global setting  -`[p]embedset global`
 
-            - In DM context:
-                1. User override - `[p]embedset user`
-                2. Global command override - `[p]embedset command global`
-                3. Global setting - `[p]embedset global`
+        - In guild context:
+          1. Channel override - `[p]embedset channel`
+          2. Server command override - `[p]embedset command server`
+          3. Server override - `[p]embedset server`
+          4. Global command override - `[p]embedset command global`
+          5. Global setting  -`[p]embedset global`
+
+        - In DM context:
+          1. User override - `[p]embedset user`
+          2. Global command override - `[p]embedset command global`
+          3. Global setting - `[p]embedset global`
```

Comment:
This currently formats as a lettered list. There is the change that in the future it will change to a numbered list so I'm leaving it as numbers.

![image](https://user-images.githubusercontent.com/51716387/234381014-ab21d759-d177-4c17-a5fd-d8c41fd4f46f.png)